Unexpected error message - The configuration of 'ExceptionMarshalling' and 'ExceptionMarshallingCustomType' on method 'Demo.MySafeHandle' is invalid.

An unexpected error message is produced with the following. I'm not using or expecting any `ExceptionMarshalling` cases.

```
...\runtime\src\samples\LibraryImportGeneratorSample\Program.cs(41,111): error SYSLIB1051: The configuration of 'ExceptionMarshalling' and 'ExceptionMarshallingCustomType' on method 'Demo.MySafeHandle' is invalid. h [...\runtime\src\samples\LibraryImportGeneratorSample\LibraryImportGeneratorSample.csproj]
...\runtime\src\samples\LibraryImportGeneratorSample\Microsoft.Interop.ComInterfaceGenerator\Microsoft.Interop.ComInterfaceGenerator\Demo.IType.cs(32,113): error CS8894: 
 Cannot use 'MySafeHandle' as a parameter type on a method attributed with 'UnmanagedCallersOnly'. [...\runtime\src\samples\LibraryImportGeneratorSample\LibraryImportGeneratorSample.csproj]
```

```csharp
namspace Demo;

public sealed class MySafeHandle : Microsoft.Win32.SafeHandles.SafeHandleZeroOrMinusOneIsInvalid
{
    public MySafeHandle()
        : base(true)
    { }
    public MySafeHandle(IntPtr handle, bool ownsHandle)
        : base(ownsHandle)
    { }
    override protected bool ReleaseHandle() {}
}
[GeneratedComInterface]
[Guid("BA0050D3-4BE1-4DFF-8130-705412C858E6")]
public partial interface IType
{
    void Foo([MarshalUsing(typeof(SafeHandleMarshaller<MySafeHandle>.ManagedToUnmanagedIn))] MySafeHandle h);
}
```

--- a/src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/GeneratorDiagnostics.cs
+++ b/src/libraries/System.Runtime.InteropServices/gen/ComInterfaceGenerator/GeneratorDiagnostics.cs
@@ -2,8 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
-using System.Collections.Generic;
-using System.Diagnostics;
 using Microsoft.CodeAnalysis;
 
 namespace Microsoft.Interop
@@ -157,7 +155,7 @@ namespace Microsoft.Interop
             new DiagnosticDescriptor(
                 Ids.TypeNotSupported,
                 GetResourceString(nameof(SR.TypeNotSupportedTitle)),
-                GetResourceString(nameof(SR.InvalidExceptionMarshallingConfigurationMessage)),
+                GetResourceString(nameof(SR.TypeNotSupportedMessageParameter)),
                 Category,
                 DiagnosticSeverity.Error,
                 isEnabledByDefault: true,

